### PR TITLE
Test Coverage

### PR DESCRIPTION
**Pull request by Taeir**
# Test coverage
- Added some tests
- Fixed some small mistakes.
- Fix sometimes failing music test

<testing, bug

### CODE REVIEW

**Comment by Diocruel** [comment]
Tests in error: 
  testSetBackgroundError(com.github.fishio.gui.TestRenderer2)
Tests run: 466, Failures: 0, Errors: 1, Skipped: 0

Build failed by a failing test.

<bug

**Comment by Taeir** [comment]
Removed tests that are incompatible with Travis.

<bug

File: src/main/java/com/github/fishio/achievements/Achievement.java [hunk 1]
```diff
@@ -87,11 +87,7 @@ public SimpleIntegerProperty getLevelProperty() {
 	
 	@Override
 	public int hashCode() {
-		final int prime = 31;
-		int result = 1;
-		
-		result = prime * result;
-		return result + achievement.hashCode();
+		return achievement.hashCode() * 31 + getLevel();
 	}
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/achievements/Achievement.java:91]: :-)
<layout

**Comment by ChielBruin** [comment]
More tests are more better!

<testing

**Comment by LMiljak** [comment]
I agree with @ChielBruin , More tests are more better! 
:+1:

Although you like to mix stuff in pull requests (like here you have testing and a bugfix that have nothing to do with each other) 

<testing, bug

**Pull request merged**